refactor/src/Cage.py

```diff
@@ -280,47 +304,80 @@ def fb_mort(days):
             :param days: number of days elapsed
             :return: fish background mortality rate
             """
-            return 0.00057 #(1000 + (days - 700)**2)/490000000
+            return 0.00057  # (1000 + (days - 700)**2)/490000000
 
         # detemine the number of fish with lice and the number of attached lice on each.
         # for now, assume there is only one TODO fix this when I understand the infestation
         # (next stage)
-        adlicepg = np.array([1] * self.num_infected_fish)/self.fish_growth_rate(days)
-        prob_lice_death = 1/(1+np.exp(-19*(adlicepg-0.63)))
-
-        ebf_death = fb_mort(days)*step_size*(self.num_fish)
-        elf_death = np.sum(prob_lice_death)*step_size
+        # adlicepg = np.repeat([1/self.fish_growth_rate(days)], (self.num_infected_fish,))
```

Comment:
Could you annotate this bit with what's the state of the method/our understanding of it?